test/blackbox-tests/test-cases/github6607.t

```diff
@@ -0,0 +1,58 @@
+This shows that the test suite is impacted by CLICOLOR_FORCE=1 (#6607).
+This environment variable should only impact the output of dune, not behavior
+visible from within the test suite.
+
+  $ cat > dune-project << EOF
+  > (lang dune 1.0)
+  > EOF
+
+  $ cat > dune << EOF
+  > (library
+  >  (name l)
+  >  (libraries stdune)
+  >  (preprocess (pps ppx_expect))
```

Comment:
The expect test should depend on the stdune and the ppx_expect packages.

Reply:
done. I added `(package stdune) (package ppx_expect)`, is that the right way to do this?

Reply:
Yup